### DISCUSSION TITLE
Impulsdauer Taster

### DISCUSSION THREAD

**Discussion by kevinhaybach** [Zefau/ioBroker.jarvis#1892] [Q&A]
Kann man die Impulsdauer des Tasters einstellen?
Ich versuche den Taster via Modbus an eine Wago Steuerung zu senden bekomme die Statusänderung aber nicht mit. Hier würde es mir wahrscheinlich enorm helfen wenn ich die Impulsdauer auf z.B. 500ms stellen könnte.

Mit freundlichen Grüßen
Kevin

**Comment by Zefau** [comment]
Impulsdauer steht aktuell auf 500ms. Kann ich aber konfigurierbar machen.

**Reply by kevinhaybach** [reply]
Ich denke 500ms klingen nach einem vernünftigen Wert und muss nicht unbedingt konfigurierbar sein.

Meine Annahme das es ein Problem mit den Zykluszeiten gibt war falsch.
Ich hatte es mit Schaltern getestet und da hat es funktioniert. Wenn ich auf Taster umgestellt habe, habe ich aber auch die Datenpunkt Eigenschaften angepasst und den eigentlichen Wert in "" geschrieben. Dadurch habe ich aber keinen Boolean sondern einen String gesendet.
